app/controllers/admin/deploy_groups_controller.rb

```diff
@@ -0,0 +1,52 @@
+class Admin::DeployGroupsController < ApplicationController
+  before_action :authorize_admin!
+  before_action :authorize_super_admin!, only: [ :create, :new, :update, :destroy ]
+  before_action :deploy_group, only: [:edit, :update, :destroy]
```

Comment:
@sandlerr yup.  If you do a `resources :deploy_groups` in your routes.rb, it creates the routes described [here](http://guides.rubyonrails.org/routing.html#crud-verbs-and-actions). Each of those methods has default behavior inherited from `ActionController::Base`.  In the case of `edit`, it looks for an `edit.html.erb` file in the appropriate views subdirectory, and renders that.